BLSQ/iaso
# IA-2716: Project Feature flag MOBILE_ENTITY_WARN_WHEN_FOUND

Adding a feature flag to DB

Related JIRA tickets : IA-2716

## Self proofreading checklist



## Changes

Added a migration

## How to test

Run migration and check feature flags in the admin, MOBILE_ENTITY_WARN_WHEN_FOUND should be there

## Print screen / video

![Screenshot 2024-03-14 at 15 17 03](https://github.com/BLSQ/iaso/assets/12494624/edf2e619-d20d-4cfc-9d36-3dc0dbf5b06c)
